src/layout.cpp

```diff
@@ -350,7 +350,11 @@ TilingResult HSFrameLeaf::computeLayout(Rectangle rect) {
             layoutResult = layoutHorizontal(rect);
             break;
     }
-    if (!smart_window_surroundings_active) {
+    if (smart_window_surroundings_active) {
+        for (auto& it : layoutResult.data) {
+            it.second.minimalDecoration = true;
+        }
+    } else { // if (!smart_window_surroundings_active)
```

Comment:
Is the commented out if statement really necessary?

Reply:
Not sure, probably not.. I can remove it

Reply:
Thanks!